float32[]  led_color
float32 led_duration
float32 led_value
